<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="frame_11">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Sliders Adjustor</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QFrame" name="frame_5">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QFrame" name="frame_3">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
                 <widget class="QLabel" name="SamplingNumber">
                  <property name="text">
                   <string>SamplingNumber</string>
                  </property>
                 </widget>
                </item>
                <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
                 <widget class="QLabel" name="NoiseNumber">
                  <property name="text">
                   <string>NoiseNumber</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_6">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QSlider" name="verticalSlider">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="verticalSlider_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_7">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Sampling Slider</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Noise Slider</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QFrame" name="frame_2">
      <property name="contextMenuPolicy">
       <enum>Qt::PreventContextMenu</enum>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QComboBox" name="comboBox"/>
       </item>
       <item>
        <widget class="QFrame" name="frame_9">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QFrame" name="frame_12">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item alignment="Qt::AlignTop">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Channel 1</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_8">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item alignment="Qt::AlignTop">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Channel 2</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_10">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item alignment="Qt::AlignTop">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Channel 3</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1250</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLoad">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuAdd_Signal">
    <property name="title">
     <string>Add Signal</string>
    </property>
    <addaction name="actionSIN"/>
    <addaction name="actionCOSINE"/>
   </widget>
   <widget class="QMenu" name="menuFrequency">
    <property name="title">
     <string>Frequency</string>
    </property>
    <addaction name="actionEnter_Frequency"/>
   </widget>
   <widget class="QMenu" name="menuMagnitude">
    <property name="title">
     <string>Magnitude</string>
    </property>
    <addaction name="actionEnter_Magnitude"/>
   </widget>
   <widget class="QMenu" name="menuRemove">
    <property name="title">
     <string>Remove</string>
    </property>
    <addaction name="actionRemove_all"/>
   </widget>
   <widget class="QMenu" name="menuSampling_Frequency">
    <property name="title">
     <string>Sampling Frequency</string>
    </property>
    <addaction name="actionHz"/>
   </widget>
   <addaction name="menuLoad"/>
   <addaction name="menuAdd_Signal"/>
   <addaction name="menuFrequency"/>
   <addaction name="menuMagnitude"/>
   <addaction name="menuRemove"/>
   <addaction name="menuSampling_Frequency"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSIN">
   <property name="text">
    <string>Sine Wave</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="actionCOSINE">
   <property name="text">
    <string>Cosine Wave </string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionFrequency">
   <property name="text">
    <string>Frequency</string>
   </property>
  </action>
  <action name="actionMagnitude">
   <property name="text">
    <string>Magnitude</string>
   </property>
  </action>
  <action name="actionChannel_1">
   <property name="text">
    <string>Channel 1</string>
   </property>
  </action>
  <action name="actionChannel_2">
   <property name="text">
    <string>Channel 2</string>
   </property>
  </action>
  <action name="actionChannel_3">
   <property name="text">
    <string>Channel 3</string>
   </property>
  </action>
  <action name="actionChannel_4">
   <property name="text">
    <string>Channel 1</string>
   </property>
  </action>
  <action name="actionChannel_5">
   <property name="text">
    <string>Channel 2</string>
   </property>
  </action>
  <action name="actionChannel_6">
   <property name="text">
    <string>Channel 3</string>
   </property>
  </action>
  <action name="actionChannel_7">
   <property name="text">
    <string>Channel 1</string>
   </property>
  </action>
  <action name="actionChannel_8">
   <property name="text">
    <string>Channel 2</string>
   </property>
  </action>
  <action name="actionChannel_9">
   <property name="text">
    <string>Channel 3</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionEnter_Frequency">
   <property name="text">
    <string>Enter Frequency</string>
   </property>
  </action>
  <action name="actionEnter_Magnitude">
   <property name="text">
    <string>Enter Magnitude</string>
   </property>
  </action>
  <action name="actionRemove_all">
   <property name="text">
    <string>Remove all</string>
   </property>
  </action>
  <action name="actionHz">
   <property name="text">
    <string>Hz</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
